near-sdk-macros/src/core_impl/utils/mod.rs

```diff
@@ -73,3 +73,24 @@ pub(crate) fn extract_vec_type(ty: &Type) -> Option<&Type> {
         _ => None,
     }
 }
+
+pub(crate) fn get_event_args(var: &syn::Variant) -> (Option<LitStr>, Option<LitStr>) {
+    let mut v: (Option<LitStr>, Option<LitStr>) = (None, None);
+    var.attrs.iter().for_each(|attr| {
+        let test = attr.parse_args_with(
+            syn::punctuated::Punctuated::<syn::MetaNameValue, syn::Token![,]>::parse_terminated,
+        );
+        if let Ok(item) = test {
```

Comment:
Shouldn't we be checking that `attr` is `event_meta` just in case something else has `standard`/`version` args?